src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts work");
```

Comment:
The message 'Truck starts work' could be more informative. According to the checklist, it should indicate what type of Machine is working, which is correct, but consider including more details about the work the Truck is doing.